packages/ui/src/theme/css/component/checkbox.scss

```diff
@@ -0,0 +1,97 @@
+.amplify-checkbox {
+  cursor: var(--amplify-components-checkbox-cursor);
+  align-items: var(--amplify-components-checkbox-align-items);
+  justify-content: var(--amplify-components-checkbox-justify-content);
+  gap: inherit;
+  &[data-disabled='true'] {
+    cursor: var(--amplify-components-checkbox-disabled-cursor);
+  }
+}
+
+.amplify-checkbox__button {
+  position: var(--amplify-components-checkbox-button-position);
+  align-items: var(--amplify-components-checkbox-button-align-items);
+  justify-content: var(--amplify-components-checkbox-button-justify-content);
+  color: var(--amplify-components-checkbox-button-color);
+
+  &::before {
+    content: var(--amplify-components-checkbox-button-before-content);
```

Comment:
This styling seems very specific to this checkbox styling implementation, do you think all of these need to be CSS variables? I don't think customers need to be able to override the `content` property.

Reply:
Oh yea, I think users wouldn't want to change the styling on `before`